### ISSUE TITLE
Easy import of multiple value tags

### ISSUE THREAD

**Issue by DaBungalow** [quodlibet/quodlibet#1800]
First off, I am absolutely loving the built in support for multiple value tags: something sorely lacking in most Linux music players.

Now my problem: I have not been able to find a way to easily import songs that have multiple value tags.  As I listen to a lot of EDM, collaborations are extremely common meaning I probably have upwards of 200 files that need to have multiple value tags. My naming structure is <albumartist>/<album>/<track> - <title> so using the split tags option on the import tags from file function will not work in my case.

Am I missing something?

**Comment by pfps** [comment]
How easy do you want this to be?  Would a solution using operon and a shell script be acceptable?

**Comment by DaBungalow** [comment]
I have never heard of operon.  Ideally, it would be integrated into the app as maybe a plugin, but for now, I would take any way that can get my music into Quod easily.

**Comment by pfps** [comment]
I agree that the built-in way of separating multiple values isn't the greatest and that being able to use a regexp as the separator would be nice.   For now you could write a script that loops over the files you want to convert, pulls out the relevant tag values using operon, rewrites // as a newline, and stuffs the value back in, again using operon.   (I tend to try scripts like this out by writing into a new tag first and also backing up my music files before any fiddling.)

**Comment by declension** [comment]
Generally it's better to use the Python Console plugin for these kinds of things, but you'd still need to know Python (and be careful..)

I've raised #1802 for an easy way to do this in the longer term.

**Comment by declension** [comment]
@DaBungalow If you're feeling brave and have backed up the songs in question, you could try this workaround in the Python console once that's enabled (select the songs to try it on, right click, _Python Console_):

   for s in sdict: s['artist'] = "\n".join([a.strip() for a in s['artist'].split('//')]); s.write()

**Comment by DaBungalow** [comment]
I believe that part of my problem was due to the fact that I didn't realize
that there is an option in Ex Falso to automatically separate multiple
tags.  When I finally found this option, I really let out a sigh of relief
as it is much easier than going through each file and manually removing the
multiple artists from the single artist tag and adding more artist tags to
add to the multiple artists.  Consequently, I am using Ex Falso.  I would
100% migrate to using Ex Falso as my tagger over Picard if the MusicBrainz
plugin would add multiple value tags correctly instead of just putting the
primary artist in the artist tag and leaving all the other artists out.

I have found a way that is manageable for me so we can close this issue and
move all future discussion to #1802.

Thanks for the response guys! I love it when devs actually listen. :)
